coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -22,11 +22,13 @@ def is_applicable(result: Result, original_file_dict, file_diff_dict):
         """
 
         if len(result.affected_code) == 0:
-            return False
+            return 'The result is not associated with any source code.'
 
         filenames = set(src.renamed_file(file_diff_dict)
                         for src in result.affected_code)
-        return any(exists(filename) for filename in filenames)
+        if any(exists(filename) for filename in filenames):
+            return True
+        return 'File(s) of code, to be ignored, do not exist.'
```

Comment:
this is confusing to read, maybe `The result is associated with source code that doesn't seem to exist.`